### PR TITLE
CORE-295 - Fix Teamspeak Exception

### PR DESCRIPTION
**Pull request by atoff**
Closes CORE-295

### CODE REVIEW

File: app/Console/Commands/TeamSpeak/TeamSpeakDaemon.php [hunk 1]
```diff
@@ -47,7 +47,13 @@ public function handle()
             try {
                 self::$connection->getAdapter()->wait();
             } catch (TeamSpeak3_Transport_Exception $e) {
-                self::$connection = $this->establishConnection();
+                try {
+                    self::$connection = $this->establishConnection();
+                } catch (TeamSpeak3_Transport_Exception $e) {
+                    // Connection failed, let the loop restart and try again
+                    $this->log('TeamSpeak connection failed: '.$e->getMessage(). '. Trying again in 15 seconds...');
```
Comment (CalumTowers) [inline app/Console/Commands/TeamSpeak/TeamSpeakDaemon.php]: Are we potentially going to miss this if it isn't working for a few days? Do we need to try three times and then throw an exception or something?

Comment (atoff) [reply app/Console/Commands/TeamSpeak/TeamSpeakDaemon.php]: Do you mean something like I just pushed?

**Comment by CalumTowers** [comment]
@atoff Can we amend to camelCase before we merge please?

File: app/Console/Commands/TeamSpeak/TeamSpeakDaemon.php [hunk 2]
```diff
@@ -41,13 +41,26 @@ class TeamSpeakDaemon extends TeamSpeakCommand
     public function handle()
     {
         self::$connection = $this->establishConnection();
+        $connection_failures = 0;
```
Comment (CalumTowers) [inline app/Console/Commands/TeamSpeak/TeamSpeakDaemon.php]: Needs amending!

Comment (atoff) [reply app/Console/Commands/TeamSpeak/TeamSpeakDaemon.php]: That's what I get for using the Git IDE :/

**Pull request merged**